Sofa/framework/Helper/src/sofa/helper/system/PluginManager.cpp

```diff
@@ -116,12 +116,26 @@ void PluginManager::readFromIniFile(const std::string& path)
 void PluginManager::writeToIniFile(const std::string& path)
 {
     std::ofstream outstream(path.c_str());
-    PluginIterator iter;
-    for( iter = m_pluginMap.begin(); iter!=m_pluginMap.end(); ++iter)
+    for( const auto& [pluginPath, _] : m_pluginMap)
```

Comment:
wahou, didn't know that syntax!

Reply:
It's called structured binding https://en.cppreference.com/w/cpp/language/structured_binding